virttest/vt_console.py

```diff
@@ -167,6 +168,11 @@ def send_ctrl(self, *args, **kwargs):
         return self.__manager.proxy_call(self.send_ctrl.__name__,
                                          *args, **kwargs)
 
+    def set_linesep(self, *args, **kwargs):
```

Comment:
better add comment to describe this function

Reply:
I don't think it's necessary, because none of func in this py file added comments, and also you can get the meaning from the name.